components/eam/src/physics/cam/hetfrz_classnuc.F90

```diff
@@ -267,7 +212,7 @@ subroutine hetfrz_classnuc_calc( &
    r_dust_a3 = hetraer(3)    
 
    ! calculate collision kernels as a function of environmental parameters and aerosol/droplet sizes
-   call collkernel(t, p, eswtr, rhwincloud, r3lx,         &
+   call collkernel(temperature, pressure, eswtr, r3lx,         &
                    r_bc,                                  &  ! BC modes
                    r_dust_a1, r_dust_a3,                  &  ! dust modes
                    Kcoll_bc,                              &  ! collision kernel [cm3 s-1]
```

Comment:
You could change 'cm3 / s-1' to 'cm^3 / s^-1' to be consistent with your notation changes elsewhere.

Reply:
changed comments to in and out for input